src/app/features/vendor/vendor-listing/listing.component.html

```diff
@@ -1 +1,35 @@
-<p>listing works!</p>
+<div class="page-container">
+  <div class="header-container">
+    <h2>All Vendors</h2>
+    <button class="add-vendor-button" [routerLink]="['/vendor/creation']">+ Add New Vendor</button>
```

Comment:
add type and id for button.
id should be unique. eg: 'add-new-vendor-btn'

Reply:
added